core/src/main/scala/kafka/server/ConfigHandler.scala

```diff
@@ -95,6 +96,19 @@ class TopicConfigHandler(private val logManager: LogManager, kafkaConfig: KafkaC
         .map(_ (0).toInt).toSeq //convert to list of partition ids
     }
   }
+  
+  def excludedConfigs(topic: String, topicConfig: Properties): Set[String] = {
+    val excludeConfigs: mutable.Set[String] = new mutable.HashSet[String]
```

Comment:
It's a bit weird how we have this `Set` that we mutate to add a single config. The previous code seemed clearer. If you wanted to transform the `Option` to a `Set`, you could simply call `toSet` on the result.